lib/capybara/screenshot/diff/test_methods.rb

```diff
@@ -56,6 +54,9 @@ def screenshot_group(name)
           FileUtils.rm_rf screenshot_dir
         end
 
+        # @param [(Symbol | String)] name
```

Comment:
What effect do these kinds of comments have?

Reply:
yeah, this was the first attempt to document and help RubyMine to validate types from the YARD doc. But then moved to rbs files. Will cleanup this for now.